{"uuid":"be3b71ad-d18e-42f9-8a10-035dc8e404fc","historyId":"4f17ab7ca9da23d432ca6873547d05b0","fullName":"AddUserTest.testAddUser","labels":[{"name":"package","value":"AddUserTest"},{"name":"testClass","value":"AddUserTest"},{"name":"testMethod","value":"testAddUser"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"D:/Automation-Practice/Orange HRM"},{"name":"subSuite","value":"AddUserTest"},{"name":"host","value":"Bunny"},{"name":"thread","value":"2428@Bunny.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testAddUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Test case 'Already_user': Expected user already exists validation expected [Already exists] but found [Should be at least 5 characters]","trace":"java.lang.AssertionError: Test case 'Already_user': Expected user already exists validation expected [Already exists] but found [Should be at least 5 characters]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat AddUserTest.testAddUser(AddUserTest.java:69)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Screenshot of testAddUser","source":"40a5d5c4-6080-4460-be2a-f61c81bc3377-attachment","type":"image/png"}],"parameters":[{"name":"arg3","value":"bhav"},{"name":"arg2","value":"l"},{"name":"arg5","value":"jhon_doe1"},{"name":"arg4","value":""},{"name":"arg1","value":"sapna"},{"name":"arg0","value":"Already_user"},{"name":"arg10","value":"Enabled"},{"name":"arg7","value":"Jhon@123"},{"name":"arg6","value":"Jhon@123"},{"name":"arg9","value":"Admin"},{"name":"arg8","value":"User_already_exists"}],"start":1756013644202,"stop":1756013647634}